scala/scala.bzl

```diff
@@ -282,6 +319,9 @@ scala_library = rule(
   implementation=_scala_library_impl,
   attrs={
       "main_class": attr.string(),
+      #TODO the implementation of java_library leads me to believe
+      # we may be able to eliminate this
+      "srcjars": attr.label_list(),
```

Comment:
ideally, this is just in srcs like the java rule:

http://bazel.io/docs/be/java.html